src/StepDefinition/MinkElementTrait.php

```diff
@@ -186,22 +186,33 @@ public function textAppearsTimesInElement($text, $element, $number, $selector =
   }
 
   /**
-   * @Then /^(?:|I )wait (?P<waitTime>\d+) ms for (?P<num>\d+) "(?P<element>[^"]*)" elements to appear?$/
-   * @Then /^(?:|I )wait for (?P<num>\d+) "(?P<element>[^"]*)" elements to appear?$/
+   * @Then I wait for :num :element elements to appear
```

Comment:
Just a note that this change somehow  fixed error when $element param was not found in tests
Example https://jenkins.ci2.drunomics.com/job/drunomics-github/job/ldp-project/view/change-requests/job/PR-198/113/console
```
And I wait for 1 ".node" elements to appear                                                                              # drunomics\BehatDrupalUtils\Context\DrupalUtilsApiContext::iWaitForElementToAppear()

      Can not find a matching value for an argument `$element` of the method `drunomics\BehatDrupalUtils\Context\DrupalUtilsDrushContext::iWaitForElementToAppear()`.
```